src/OrleansRuntime/LogViews/PrimaryBasedLogViewAdaptor.cs

```diff
@@ -317,7 +317,11 @@ protected bool IsMyClusterJoined()
 
         #region Interface
 
-        public void Submit(TLogEntry logentry)
+        /// <summary>
```

Comment:
Use <inheritdoc /> wherever possible, many of these methods already have carefully written descriptions (see ILogViewStorageInterface.cs)

If necessary use cref="ILogViewRead{TView,TEntry}" or cref="ILogViewUpdate{TView,TEntry}"